src/application/scene/Todo.java

```diff
@@ -1,14 +1,186 @@
+    @FXML
+    private DatePicker datePicker;
+    @FXML
+    private ListView<String> listView_TODO;
+    @FXML
+    private ListView<String> listView_DOING;
+    @FXML
+    private ListView<String> listView_DONE;
+    @FXML
+    private ChoiceBox<String> choiceBox_status;
+
+
+    public void initialize() {
+
+        textField_eventName.setPromptText("TODO Events");
+
+        datePicker.setValue(LocalDate.now());
+
+        //Add TODO DOING DONE status to the choiceBox dropdown
+        choiceBox_status.getItems().addAll("TODO", "DOING", "DONE");
+        choiceBox_status.setValue("TODO");
+
+        getHistory(listView_TODO, "TODO");
+        getHistory(listView_DOING, "DOING");
+        getHistory(listView_DONE, "DONE");
+
+
+    }
+
+    @FXML
+    private void addEvent() {
+        String status = choiceBox_status.getValue();
+        String eventMessage = datePicker.getValue() + " " + textField_eventName.getText();
+
+        if (textField_eventName.getText().isEmpty()) {
+            noContentAlert();
+        } else {
+            switch (status) {
+                case "TODO":
+                    addToList(listView_TODO, "TODO", eventMessage);
+                    break;
+                case "DOING":
+                    addToList(listView_DOING, "TODO", eventMessage);
+                    break;
+                case "DONE":
+                    addToList(listView_DONE, "TODO", eventMessage);
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+
+
+    @FXML
+    private void moveToDOING() {
+        String currentItem = listView_TODO.getSelectionModel().getSelectedItem();
+        if (!currentItem.isEmpty()) {
+            listView_DOING.getItems().add(currentItem);
+            listView_TODO.getItems().remove(currentItem);
+
+            rewriteHistory(listView_TODO, "TODO");
+            rewriteHistory(listView_DOING, "DOING");
+        }
+    }
+
+    @FXML
+    private void moveToDONE() {
+        String currentItem = listView_DOING.getSelectionModel().getSelectedItem();
+        if (!currentItem.isEmpty()) {
+            listView_DONE.getItems().add(currentItem);
+            listView_DOING.getItems().remove(currentItem);
+
+            rewriteHistory(listView_DOING, "DOING");
+            rewriteHistory(listView_DONE, "DONE");
+        }
+    }
+
+    @FXML
+    private void clearDONE() {
+        Alert alert_clearDONEConfirmation = new Alert(Alert.AlertType.CONFIRMATION, "Do you really want to remove all Events in this area", ButtonType.YES, ButtonType.NO);
+        ButtonType result = alert_clearDONEConfirmation.showAndWait().orElse(ButtonType.NO);
+
+        if (ButtonType.YES.equals(result)) {
+            listView_DONE.getItems().clear();
+        }
+
+        try {
+            File file = new File("./src/application/scene/ToDoLists/DONE.txt");
+            FileWriter clearWriter = new FileWriter(file); //the true will append the new data
```

Comment:
This comment is confusing. Could it be rewritten to be more clear (or removed entirely)

Reply:
Removed